### DISCUSSION TITLE
No drag events on buttons?

### DISCUSSION THREAD

**Discussion by herronelou** [themesberg/flowbite-svelte#1301] [Q&A]
Hello,
I've been trying to implement a drag and drop feature to re-organize elements in my app, and have struggled for a bit trying to do it with flowbite-svelte components, as none of the events seemed to trigger (dragstart, drag, dragend, dragover, etc..).

Upon replacing my Flowbite Buttons with regular buttons, everything worked as expected., is it expected that Drag and Drop isn't supported on Flowbite components?

**Comment by shinokada** [comment]
1. Can you list the components that you need event forwarding.
2. Can you list the events you need.

**Reply by herronelou** [reply]
Thanks for your reply.

My case was very specific, needing `dragstart`, `drag` and `dragend` on `Button`, `dragover` on `Li`.
But I guess I'm wondering if there's a generic way I could access any event from any component, or if it requires modifying the components to forward the events.

I've made something like this:

https://github.com/themesberg/flowbite-svelte/assets/1182452/07800a7f-47d1-414a-901d-b237910e961c

The grey buttons aren't `<Button>`, they're good old `<button>` that I styled to look like a flowbite button.
